### ISSUE TITLE
It should be easy to tweet links as they are curated for Argo

### ISSUE THREAD

**Issue by aschweigert** [INN/link-roundups#27]
@abehandler said:

If a reporter has gone through the trouble of adding a link to Argo, then it should be very easy to send the link out on Twitter as well. The reporter should have the ability to tweet the link automatically (there would be a setting for this). That way the news organization can get the most out of (extremely limited) reporter time.

Argo Links would not necessarily need to be responsible for tweeting the links. It could simply export the links to a Buffer queue automatically (if Buffer has an API). Alternately, you could build an independent, external application that read from the XML feed of argo links and fed it into an auto Tweeting application.

@gabestein and I started working on something like this at The Lens but gave up once we lost our (excellent) news roundup writer. We modified the XML feed of Argo links to include a 140 character tweet that included a bitly version of the link. We wanted to include a character counter in the argolinks bookmarklet (on advice of @myersnews) but never got there.

We tried to get the links going out on Facebook as well but found that Facebook tended to bury posts that were added automatically with the API.

Moved from: https://github.com/INN/argo-links/issues/26

**Comment by aschweigert** [comment]
Also had a request for this from Aspen Journalism.

tl;dr of the above: when i save a link using the browser bookmarklet, at the same time allow me to post the link to twitter/fb. Likely just used the headline for twitter and let people modify it when you pop the tweet window, use the headline, desc and photo for FB.

A possible enhancement might be to do something with buffer but that doesn't really seem as essential (and not something every publisher would want anyway).
